hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin/nssummary/DiskUsageSubCommand.java

```diff
@@ -101,21 +99,21 @@ public Void call() throws Exception {
       return null;
     }
 
-    HashMap<String, Object> duResponse = getResponseMap(response);
+    ObjectMapper objectMapper = new ObjectMapper();
+    JsonNode duResponse = objectMapper.readTree(response);
```

Comment:
Can we use here also 
```suggestion
    HashMap<String, Object> responseMap = getResponseMap(response);
```